### ISSUE TITLE
Invalid JS code when creating new jQuery elements

### ISSUE THREAD

**Issue by CyberStarLight** [bridgedotnet/Bridge#3075]
When trying to create a new element using the constructor `new jQuery(ElementType elementType)`

for example: `jQuery h = new jQuery(ElementType.Button);`

It creates an error in the JS code because it writes an invalid string with an extra double quote inside it, like this: `var h = $("<"button>");`
notice the extra `"` before `button`.

### Steps To Reproduce
just add the line:

```csharp
jQuery h = new jQuery(ElementType.Button);
```

anywhere in the project.

https://deck.net/b775d2ebe61433578ddac71a564da491

```c#
public class Program
{
    public static void Main()
    {
        jQuery h = new jQuery(ElementType.Button);
    }
}
```

### Expected Result

```js
var h = $("<button>");
```

### Actual Result

```js
var h = $("<"button>");
```

**Comment by geoffreymcgill** [comment]
Hi @CyberStarLight 

Thanks for the excellent report and for bringing this defect to our attention. 

I have verified the issue and we should be able to fix for the 16.3 release.

Here's a test case that reproduces the issue independent of **Bridge.jQuery**.

https://deck.net/6d414ce9833f1e3ff86313797c87c768

```csharp
public class Program
{
        public static void Main(string[] args)
        {
            Test(SomeType.Apple);
        }

        [Template("{0:raw}")]
        public static void Test(SomeType elementType) { }
}

[Enum(Emit.StringNamePreserveCase)]
public enum SomeType
{
	Apple
}
```

**Comment by vladsch** [comment]
As a workaround, you can disable source map generation. We are working on it...
